lib/train/transports/ssh.rb

```diff
@@ -108,6 +113,18 @@ def validate_options(options)
       self
     end
 
+    # Creates an SSH Authentication KeyManager instance and saves it for
+    # potential future reuse.
+    #
+    # @return [Hash] hash of SSH Known Identities
+    # @api private
+    def ssh_known_identities
+      return @manager.known_identities if @manager
+      # Force KeyManager to load the key(s)
+      @manager = Net::SSH::Authentication::KeyManager.new(nil?).each_identity {}
```

Comment:
hmm interesting. Can you fill me in here -- what is `nil?`? It seems to resolve to `false`,

```
irb(main):004:0> nil?
=> false
```
or am I missing something?